assets/css/edd-admin.css

```diff
@@ -1825,7 +1817,7 @@ input[class*="edd-price-field"] {
 .edd-admin-box-inside {
 	border-bottom: 1px solid #f1f1f1;
 	clear: both;
-	padding: 10px 12px;
+	padding: 12px 12px;
```

Comment:
Not a big deal, but since these are identical, you can just set this to `padding: 12px;`.

Reply:
Might as well make it more concise.